src/commands/update.js

```diff
@@ -6,6 +6,7 @@ const cmd = {
   alias: ['u'],
   run: async toolbox => {
     const generators = (await generator(toolbox)).update
+    await exec('npx eslint \"**/*.{js,jsx}\" --fix')
```

Comment:
we need to check if exists an eslint file, if we run it without an eslint file, the `npx eslit` will ask to install and configure it.
it could generate a bad DX for the programmer